examples/gaussian_process/plot_gpr_dim_selection.py

```diff
@@ -0,0 +1,110 @@
+"""
+=========================================================
+Illustration of SelectDimensionKernel
+=========================================================
+A simple two-dimensional regression example computed in two different ways:
+1. With a product of two RBF kernels on each feature.
+2. With one anisotropic RBF kernels applied on both feature.
+
+The figures illustrate the property of SelectDimensionKernel when applied on
+different features of input data.
+"""
+print(__doc__)
+
+
+# Authors: Behzad Tabibian <me@btabibian.com>
+#
+# License: BSD 3 clause
+
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.gaussian_process.kernels import SelectDimensionKernel, RBF
+
+# Fixing seed
+np.random.seed(1)
+
+# Define a kernel with sum of two RBF kernels applied on individual dimentsion.
+kernel = SelectDimensionKernel(RBF(length_scale=0.1), np.array([0])) * \
+         SelectDimensionKernel(RBF(length_scale=0.4), np.array([1]))
```

Comment:
Sorry, if this is obvious but what is the difference between doing this and using the kernel below?

For both the kernel remains the same and hence the negative log-likelihood should also be the same?